artifactory/commands/helm/utils.go

```
@@ -117,3 +117,66 @@ type ociReference struct {
 	Repository string
 	Reference  string
 }
+
+func getPaths(helmArgs []string) []string {
+	var paths []string
+	for _, arg := range helmArgs {
+		if !strings.HasPrefix(arg, "-") {
+			paths = append(paths, arg)
+		}
+	}
+	return paths
+}
+
+func removeDuplicateDependencies(buildInfo *entities.BuildInfo) {
+	if buildInfo == nil {
+		return
+	}
+	for moduleIdx, module := range buildInfo.Modules {
+		dependenciesMap := make(map[string]entities.Dependency)
+		var dependencies []entities.Dependency
+		for _, dependency := range module.Dependencies {
+			sha256 := dependency.Sha256
+			_, exist := dependenciesMap[sha256]
+			if sha256 != "" && !exist {
```

Dependencies without SHA256 are silently dropped. 
Should either:
**Log a warning**: `log.Warn("Dependency", dependency.Id, "has no SHA256, excluding from build info")`

---

Addressed it.